js/src/modals/CreateAccount/store.js

```diff
@@ -220,7 +226,28 @@ export default class Store {
     this.stage--;
   }
 
-  createAccount = () => {
+  createAccount = (vaultStore) => {
```

Comment:
Wouldn't it make sense to attach the `vaultStore` to this Store before, instead of passing this as an argument (might be used for other methods)?